File: jihungram_android/app/src/main/java/com/example/jihun/jihungram/Article.java
package com.example.jihun.jihungram;

/**
 * Created by jihun on 2016-07-17.
 */
public class Article {
    private int articleNumber;
    private String title;
    private String writer;
    private String id;
    private String content;
    private String writeDate;
    private String imgName;
    private int hits;

    public Article(int articleNumber, String title, String writer, String id, String content, String writeDate, String imgName, int hits) {
        this.articleNumber = articleNumber;
        this.title = title;
        this.writer = writer;
        this.id = id;
        this.content = content;
        this.writeDate = writeDate;
        this.imgName = imgName;
        this.hits = hits;
    }

    public int getArticleNumber() {
        return articleNumber;
    }

    public String getTitle() {
        return title;
    }

    public String getWriter() {
        return writer;
    }

    public String getId() {
        return id;
    }

    public String getContent() {
        return content;
    }

    public String getWriteDate() {
        return writeDate;
    }

    public String getImgName() {
        return imgName;
    }

    public int getHits() { return hits; }
}

File: jihungram_server/app/model/article.py
# -*- coding: utf-8 -*-
from app.database import DBManager
db = DBManager.db
class Article(db.Model):
    articleNumber = db.Column(db.Integer, unique=True, primary_key=True)
    title = db.Column(db.String(200))
    writer = db.Column(db.String(80))
    id = db.Column(db.String(200))
    content = db.Column(db.String(1000))
    writeDate = db.Column(db.String(50))
    imgName = db.Column(db.String(100))
    hits = db.Column(db.Integer)

    def __init__(self, title, writer, id, content, writeDate, imgName):
        self.title = title
        self.writer = writer
        self.id = id
        self.content = content
        self.writeDate = writeDate
        self.imgName = imgName
        self.hits = 0

